srsue/src/stack/upper/nas.cc

```diff
@@ -1272,6 +1273,14 @@ void nas::parse_service_reject(uint32_t lcid, unique_byte_buffer_t pdu)
   }
 
   srsran::console("Received service reject with EMM cause=0x%x.\n", service_reject.emm_cause);
+
+
+// do not accept if the message is not protected when the EMM cause is #25 (TS 24.301 Sec. 4.4.4.2)
```

Comment:
remove first empty line and fix indent please

Reply:
Yes, fixed!